core/src/extended_rpc.rs

```diff
@@ -271,15 +153,6 @@ where
         self.client.get_raw_transaction(txid, block_hash)
     }
 
-    #[tracing::instrument(skip(self), err(level = tracing::Level::ERROR), ret(level = tracing::Level::TRACE))]
-    pub fn get_transaction(
```

Comment:
Why did we remove these functions? These functions do not need test since we did not implement them.

Reply:
They weren't used. @ekrembal said we should delete the functions we don't use.